checks.go

```diff
@@ -0,0 +1,24 @@
+package datadog
+
+type Check struct {
+	Check     string   `json:"check"`
+	HostName  string   `json:"host_name"`
+	Status    status   `json:"status"`
+	Timestamp string   `json:"timestamp,omitempty"`
+	Message   string   `json:"message,omitempty"`
+	Tags      []string `json:"tags,omitempty"`
+}
+
+type status int
+
+const OK status = 0
```

Comment:
I agree that's idiomatic, but in this case OK must always be exactly 0, WARNING must be 1, etc. Does `iota` guarantee it will assign counting from 0?